en/mozorg/about/governance/policies/reporting.ftl

```diff
@@ -0,0 +1,52 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/about/governance/policies/participation/reporting/
+
+reporting-community-participation = Community Participation Guidelines - How to Report
+reporting-how-to-report-violations = How to Report Violations of the Community Participation Guidelines
+reporting-this-document-provides = This document provides high-level information, for understanding and reporting violations of { -brand-name-mozilla }'s Community Participation Guidelines.
+reporting-from-the-community-participation = From the <a href="{ $cpg }">Community Participation Guidelines</a>:
+reporting-the-heart-of-mozilla-is = The heart of { -brand-name-mozilla } is people. We put people first and do our best to recognize, appreciate and respect the diversity of our global contributors. The { -brand-name-mozilla } Project welcomes contributions from everyone who shares our goals and wants to contribute in a healthy and constructive manner within our community. As such, we have adopted this code of conduct and require all those who participate to agree and adhere to these Community Participation Guidelines in order to help us create a safe and positive community experience for all.
+reporting-this-document-is-intended = This document is intended as an interface to existing documents, processes and people responsible for ensuring { -brand-name-mozilla }’s communities are healthy, and inclusive for all.
+reporting-when-to-report = When To Report
+reporting-please-report-all-incidents = Please report all incidents where someone has engaged in behavior that is potentially illegal or makes you or someone else feel unsafe, unwelcome or uncomfortable <a href="{ $cpg }">as further explained in the CPG</a>.
+reporting-how-to-give-a-report = How to Give a Report
+reporting-if-you-believe-someone = If you believe someone is in physical danger call your local emergency number.
+reporting-if-you-have-a-report-by = If you have a report <strong>by <em>and</em> about</strong> a contributor (for example, the report is made <strong>by</strong> one contributor <strong>about</strong> another contributor), then you should make your report at the <a href="{ $community_hotline }">Community Participation Guidelines hotline</a>.
+reporting-if-you-have-a-report-involving = If you have a report <strong>involving an employee, contractor, or vendor</strong> (for example, the report is made <strong>by</strong> an employee or is <strong>about</strong> an employee) then you should report at the <a href="{ $employee_hotline }">{ -brand-name-mozilla } Employee hotline</a>.
```

Comment:
Add a comma after `an employee` so it reads `... <strong>about</strong> an employee), then you should...`